style.css

```diff
@@ -1,5 +1,5 @@
 h1 {
-  font-family: Arial, Helvetica, sans-serif;
+  font-family: "Comic Sans";
   font-size: 40px;
```

Comment:
Why are you still using px? It's 2017...

Reply:
Because I'm old-school and I don't know any other values than pixels. What would you use?

Reply:
Use `ch`.

Reply:
Przepraszam

Reply:
True greatness and site responsiveness can be achieved only by using `vw`.